src/system.cpp

```diff
@@ -86,7 +86,7 @@ static int SYS_initLEDs(void)
 
 void SYS_chargerTask(void)
 {
-    bool isCharging = ~digitalRead(STAT_PIN);
+    bool isCharging = System.batteryState() == BATTERY_STATE_CHARGING ? true : false;
```

Comment:
This is very difficult to read - why not simply do `(System.batteryState() == BATTERY_STATE_CHARGING)`?